patches/abstractions-xunit/0001-Patching.patch

```diff
@@ -0,0 +1,61 @@
+From b68777acde280b91039b98ff36c4d028064a8f52 Mon Sep 17 00:00:00 2001
+From: Matt Mitchell <mmitche@microsoft.com>
+Date: Wed, 19 Jul 2023 11:47:26 -0700
+Subject: [PATCH] Patching
+
+---
+ global.json                                       |  6 ------
+ src/xunit.abstractions/Properties/AssemblyInfo.cs |  2 +-
+ src/xunit.abstractions/xunit.abstractions.csproj  | 11 +++--------
+ 3 files changed, 4 insertions(+), 15 deletions(-)
+ delete mode 100644 global.json
+
+diff --git a/global.json b/global.json
+deleted file mode 100644
+index 9f78419..0000000
+--- a/global.json
++++ /dev/null
+@@ -1,6 +0,0 @@
+-{
+-  "sdk": {
+-    "version": "6.0.0",
+-    "rollForward": "latestMinor"
+-  }
+-}
+diff --git a/src/xunit.abstractions/Properties/AssemblyInfo.cs b/src/xunit.abstractions/Properties/AssemblyInfo.cs
+index 041f22b..c5b5124 100644
+--- a/src/xunit.abstractions/Properties/AssemblyInfo.cs
++++ b/src/xunit.abstractions/Properties/AssemblyInfo.cs
+@@ -17,6 +17,6 @@
+ [assembly: AssemblyProduct("xUnit.net Testing Framework")]
+ [assembly: AssemblyCopyright("Copyright (C) .NET Foundation")]
+ [assembly: CLSCompliant(true)]
+-[assembly: AssemblyVersion("2.0.1.0")]
++[assembly: AssemblyVersion("2.0.3.0")]
```

Comment:
Why a different version here?

Reply:
Back when I first added this repo, the github repo did not have a tag pushed for the 2.0.3 version of the package. The source code is apparently the same though. Let me re-check.

Reply:
https://github.com/xunit/abstractions.xunit/blob/2.0.3/src/xunit.abstractions/Properties/AssemblyInfo.cs#L20

Yeah, he pushed another tag but it's just the 2.0.1 tag again. So this is required.